src/UI/MainMenuOptions.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include "UI.h"
+
+class MainMenuOptions : public Window
+{
+public:
+    MainMenuOptions();
+    virtual ~MainMenuOptions();
+
+    virtual void update();
+
+protected:
+    virtual void init();
+    void visibilityChanged(bool visible) override;
```

Comment:
Clang does all or nothing warnings for use of `override`. Once you use it on one method, you need to also use it on all other overridden virtual methods to avoid warnings.

In particular, the `update` methods is warned about by Clang.